hair.html

```diff
@@ -7,19 +7,59 @@
     <link rel="stylesheet" href="./hair-style.css">
 </head>
 <body>
-  <div class="header">
-    <a href="hair.html" class="logo">
-      <img src="./Images/Hair-Logo.jpg" alt="">
-    </a>
-    <ul class="nav">
-      <li><a href="">Home</a></li>
-      <li><a href="">About</a></li>
-      <li><a href="">Gallery</a></li>
-      <li><a href="">Blog</a></li>
-      <li><a href="">Contact</a></li>
-    </ul>
+  <div class= main-container>
+    <div class="header">
+      <a href="hair.html" class="logo">
+        <img src="./Images/Hair-Logo.jpg" alt="">
+      </a>
+      <ul class="nav">
+        <li><a href="">Home</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Gallery</a></li>
+        <li><a href="">Blog</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
     
-  </div>
-  <div class="body">
+    </div>
+    <div class="body">
+      <img class="main-img" src="./Images/hair-salon.jpg" alt="">
```

Comment:
i'd but this in css as the background of .body class. 
`background-image: url('./Images/hair-salon.jpg')`